optional-integration-analysis/perform-integration.R

```diff
@@ -89,9 +102,21 @@ if(!("library_id" %in% colnames(colData(merged_sce)))){
 
 # Perform integration with specified method
 if ("fastMNN" %in% integration_methods) {
+  # Format `fastmnn_merge_order` if provided; can only be used when auto.merge is FALSE
+  if(is.null(opt$fastmnn_auto_merge)) {
```

Comment:
```suggestion
  if(!opt$fastmnn_auto_merge) {
```
Since the default is set to false, I think you should be able to check for false rather than it being null.